### PR TITLE
Ups alpha.2 release

### PR DESCRIPTION
**Pull request by matzew**
@cvasilak @lfryc can you review and merge ?

### CODE REVIEW

**Comment by lfryc** [comment]
The list of highlights should be a bit better formatted so that user can go through the list of changes quickly, without reading whole paragraph. But it is important in Beta/CR/Final releases, where we can repeat that.

typo: `wizzards` > `wizards`

**Comment by matzew** [comment]
for a full detailed list, there is link to JIRA. I felt it looks odd, to just have two/three bullet points .

**Comment by lfryc** [comment]
Yea, the reason is that people will open a link from e.g. Twitter and if it isn't properly formatted, they may just close it without reading. But as I said, not important for Alpha release.

File: _posts/2015-05-04-aerogear-unified-push-alpha2-is-out.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+layout: news
+section: news
+title: AeroGear UnifiedPush Server 1.1.0-alpha.2 released!
+author: matzew
+module: push
+releases: 1.1.0-alpha.2
+---
+
+
+We are happy to announce that the [Unified Push Server 1.1.0-alpha.2](https://github.com/aerogear/aerogear-unifiedpush-server/releases/tag/1.1.0-alpha.2) has been released !
```
Comment (cvasilak) [inline _posts/2015-05-04-aerogear-unified-push-alpha2-is-out.md]: the "description" title in the GH release notes page use ```1.0.0-alpha.2```. We should change to ```1.1.0-alpha.2```

Comment (matzew) [inline _posts/2015-05-04-aerogear-unified-push-alpha2-is-out.md]: cghabged that

On Mon, May 4, 2015 at 12:43 PM, Christos Vasilakis <
notifications@github.com> wrote:

> In _posts/2015-05-04-aerogear-unified-push-alpha2-is-out.md
> <https://github.com/aerogear/aerogear.org/pull/582#discussion_r29576902>:
>
> > @@ -0,0 +1,19 @@
> > +---
> > +layout: news
> > +section: news
> > +title: AeroGear UnifiedPush Server 1.1.0-alpha.2 released!
> > +author: matzew
> > +module: push
> > +releases: 1.1.0-alpha.2
> > +---
> > +
> > +
> > +We are happy to announce that the [Unified Push Server 1.1.0-alpha.2](https://github.com/aerogear/aerogear-unifiedpush-server/releases/tag/1.1.0-alpha.2) has been released !
>
> the "description" title in the GH release notes page use 1.0.0-alpha.2.
> We should change to 1.1.0-alpha.2
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/582/files#r29576902>.
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Comment by cvasilak** [comment]
looks good :+1: 

@lfryc mind if I merge?

**Comment by matzew** [comment]
+1 :-)

On Mon, May 4, 2015 at 12:51 PM, Christos Vasilakis <
notifications@github.com> wrote:

> looks good [image: :+1:]
>
> @lfryc <https://github.com/lfryc> mind if I merge?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/582#issuecomment-98674708>.
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Comment by lfryc** [comment]
go ahead @cvasilak !

**Pull request merged**